server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +213,83 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+
+	try {
```

Comment:
You should 100% check the provided variables before actually storing anything in the DB--it's completely possible for someone to send a request not from your webpage, and we don't want to willy-nilly accept those values--we should validate that the values are what we expect